WeakAurasArchive/WeakAurasArchive.toc

```diff
@@ -0,0 +1,7 @@
+## Interface: 80205
```

Comment:
80300

Reply:
We should do like all the other weakauras "addons"
```lua
#@retail@
## Interface: 80300
#@end-retail@
#@non-retail@
# ## Interface: 11303
#@end-non-retail@
```